src/docs/product/sentry-basics/integrate-frontend/configure-scms.mdx

```diff
@@ -13,110 +13,212 @@ Sentry uses commit metadata from your source code repositories to help you resol
 
    ![Add code mapping](configure-scms-09.png)
 
-## Step 2: Set commit tracking
+### UI Walkthrough
 
-Now that you've set up releases in Sentry as part of your CI/CD process and integrated your source code repositories, you can associate commits from your linked repository to your releases.
+Follow along with the animated gif below to see how to integrate your GitHub account and repository.
 
-<Note>
+<div style="position: relative; padding-bottom: calc(48.66707997520149% + 41px); height: 0; width: 100%">
+  <iframe
+    src="https://demo.arcade.software/OlEYtzImezSBwsV8SVc8?embed"
+    frameborder="0"
+    loading="lazy"
+    webkitallowfullscreen
+    mozallowfullscreen
+    allowfullscreen
+    style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;color-scheme: light;"
+    title="FE Tutorial -> Install GitHub Config & Add Code Mapping"
+  ></iframe>
+</div>
 
-In the demo project, we use a `Makefile` to handle our build-related tasks. If you are not using the provided React demo code and do not have a `Makefile`, you can run the `sentry-cli` commands used in this tutorial directly from the `command line` or integrate the commands into the relevant build script.
+## 2. Set Up Commit Tracking
 
-</Note>
+A _release_ is a version of your code deployed to an environment. See the [Releases](/product/releases/) documentation to learn more. Sentry uses releases to match up your source maps and source code. Releases can also be used to enable Sentry's [Suspect Commits](/product/issues/suspect-commits/) feature.
 
-1. Open the `Makefile` in your project.
+The Sentry Webpack plugin can automatically configure release information and send it to Sentry when you create a new build of your project. Let's enable this feature.
 
-2. Add the following target at the bottom of the file:
+1. Go back to your `frontend-tutorial` project and stop it with `Ctrl + C`.
 
-   ```bash
-   associate_commits:
-       sentry-cli releases -o $(SENTRY_ORG) -p $(SENTRY_PROJECT) set-commits --auto $(REACT_APP_RELEASE_VERSION)
+1. To start associating commits and releases, add the following lines of code to `webpack.config.js` in your Sentry Webpack plugin config:
+
+   ```javascript {filename:webpack.config.js}
+      sentryWebpackPlugin({
+         org: "<your_org>",
+         project: "<your_project>",
+
+         // Auth tokens can be obtained by creating an internal integration
+         // at https://<organiaztion_id>.sentry.io/settings/developer-settings/
+         // and need "Release: Admin" and "Organization: Read & Write" permissions
+         authToken: process.env.SENTRY_AUTH_TOKEN,
+
+         // Enable automatically creating releases and associating commits
+         release: {
+         create: true,
+         setCommits: {
+            auto: true,
+         },
+         },
```

Comment:
Hey @lforst - what's the default value here? I had to set `setCommits.auto = true` like this for commits to be associated, but it feels like it should be the default.

Looks like it's false according to docs, but not super super clear: https://www.npmjs.com/package/@sentry/webpack-plugin#releasesetcommits